apps/nspanel-lovelance-ui/nspanel-lovelance-ui.py

```diff
@@ -33,8 +33,8 @@ def __init__(self, api, config):
     self.update_date("")
 
     # Setup weather callback - send/update weather information every 10 minutes
-    time = datetime.time(0, 0, 0)
-    self.api.run_every(self.update_screensaver_weather, "now", 10 * 60)
+    weather_interval = 15 * 60 # 15 minutes
+    self.api.run_every(self.update_screensaver_weather, "now", weather_interval)
 
     self.update_screensaver_weather("")
```

Comment:
Is this still needed? The function gets called in the run_every command now and then every 15 minutes.

Reply:
no it isn't, will remove it, sorry for the mess I created, should have left my fingers from the weather stuff :D